### ISSUE TITLE
Disable /usr/lib/trilium/trilium --type=renderer --field-trial-handle ?

### ISSUE THREAD

**Issue by gerroon** [zadam/trilium#780]
HI

Is it possible to disable this field trial stuff with the Trilium releases? This is one reason I stopped using Chromium since the regular releases are always shipped with that.

I am using Trilium to take notes not to be a guinea pig for Google if you will.

**Comment by zadam** [comment]
I don't think this is too relevant for electron since google is probably not doing field trials there ...

But I could not really find information related to the electron specifically.
